From 8f2f7845341dde2ec4184e531d05432e79e9936b Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Thu, 27 Nov 2014 15:16:39 +0100
Subject: [PATCH 2/2] lxc-alpine: create a default tty for console

Create a tty so we get login prompt on console by default

Signed-off-by: Natanael Copa <ncopa@alpinelinux.org>
---
 templates/lxc-alpine.in | 1 +
 1 file changed, 1 insertion(+)

diff --git a/templates/lxc-alpine.in b/templates/lxc-alpine.in
index 47df559..ba27aea 100644
--- a/templates/lxc-alpine.in
+++ b/templates/lxc-alpine.in
@@ -121,6 +121,7 @@ configure_alpine() {
     cat >"$rootfs"/etc/inittab<<EOF
 ::sysinit:/sbin/rc sysinit
 ::wait:/sbin/rc default
+console:12345:respawn:/sbin/getty 38400 console
 tty1:12345:respawn:/sbin/getty 38400 tty1
 tty2:12345:respawn:/sbin/getty 38400 tty2
 tty3:12345:respawn:/sbin/getty 38400 tty3
-- 
2.1.3

